drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -871,6 +871,7 @@ function DragAndDropBlock(runtime, element, configuration) {
                 input: input,
                 displayName: item.displayName,
                 imageURL: imageURL,
```

Comment:
Do we need to keep both `imageURL` and `expandedImageURL` as separate attributes of `itemProperties` in this view? Seems like a potential source of bugs. You could probably get away with changing line 860 to

```
var imageURL = item.expandedImageURL || item.backgroundImage;  // Fall back on "backgroundImage" to be backward-compatible
```
and then just using `imageURL` everywhere from that point on.